packages/legend-application/src/components/WebApplicationClientSideStorageProviderTestUtils.tsx

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import { WebApplicationClientSideStorage } from '../stores/WebApplicationClientSideStorage.js';
+import { createMock } from '@finos/legend-shared';
+
+export const TEST__provideMockedWebApplicationClientSideStorage =
+  (customization?: {
+    mock?: WebApplicationClientSideStorage;
+  }): WebApplicationClientSideStorage => {
+    const value = customization?.mock ?? new WebApplicationClientSideStorage();
+    const MockWebApplicationClientSideStorageProvider = require('./WebApplicationClientSideStorageProvider.js'); // eslint-disable-line @typescript-eslint/no-unsafe-assignment
```

Comment:
Maybe could add disable eslint 'next-line' for shorter line length? `// eslint-disable-next-line @typescript-eslint/no-unsafe-assignment`